src/Devices/Devices.js

```diff
@@ -222,8 +223,8 @@ const DevicesComponent = ({
           <Grid item xs={12} md={12} lg={12} xl={12}>
             <Typography>
               To rename a device, use the pencil icon <Edit fontSize="small" />. Add your farm code
-              in all caps anywhere in the nickname, plus any other info you want. Example: &quot;ABC
-              (Farmer Brown top of the hill)&quot;
+              in all caps anywhere in the nickname, plus any other info you want. Avoid using captial letters in the 
+              name other than your farm code. 
```

Comment:
Can we go back to including the example string, as well as saying "multiple capital letters" instead of "captial" (note the spelling) like this has?